labtool2.0/src/reducers/coursePageLogicReducer.js

```diff
@@ -230,4 +233,12 @@ export const toggleCodeReview = reviewNumber => {
   }
 }
 
+export const toggleMassAssign = () => {
+  return async dispatch => {
+    dispatch({
+      type: 'COURSE_PAGE_TOGGLE_MASS_ASSIGN'
+    })
+  }
+}
+
```

Comment:
Where is this function used?

Reply:
This is an old leftover that can be safely removed